models/nomad/ethereum/nomad_ethereum_view_bridge_transactions.sql

```diff
@@ -1,7 +1,17 @@
 {{
   config(
-        alias='view_bridge_transactions'
-  )
+      alias='view_bridge_transactions',
+      post_hook = "
```

Comment:
very nice thanks for trying it! I think we should make this block into a macro, this would hopefully prevent typos and other smaller issues. Will also help if we ever need to change a bunch of things.

Reply:
makes sense, I'll give it a try

Reply:
@aalan3 Here's what I got so far, the only thing that's still bothering me is the backslash that's needed to break double quotes but otherwise wdyt ?